src/SFA.DAS.EAS.Portal/Application/Commands/Paye/PayeSchemeAddedCommandHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using SFA.DAS.EAS.Portal.Application.AccountHelper;
+using SFA.DAS.EAS.Portal.Application.Services;
+using SFA.DAS.EAS.Portal.Client.Types;
+
+namespace SFA.DAS.EAS.Portal.Application.Commands.Paye
+{
+    public class PayeSchemeAddedCommandHandler : ICommandHandler<PayeSchemeAddedCommand>
```

Comment:
The `Command` and `CommandHandler` could be combined into a single `Command`, as per the pattern in CON-409.

Reply:
No harm in leaving as current. Comes down to preference mainly. If there is a refactor of system then it should be done as tech debt down the line

Reply:
And it follows the DAS standard so will be consistent across the service.